improver/api/__init__.py

```diff
@@ -0,0 +1,131 @@
+# (C) Crown copyright, Met Office. All rights reserved.
```

Comment:
As I commented in https://github.com/metoppv/improver/pull/2013 - the "`copyright`" is "`Copyright`" in the `copyright_check` script, so if this file were to be checked with your new checker script it would fail on the copyright notice text.

Reply:
Good spotting.  Yep, we need #2013 merged soon to avoid this sorta thing.